src/parsec_service/install_parsec_linux.md

```diff
@@ -12,54 +12,82 @@ can be a space or comma-separated subset of: `mbed-crypto-provider`, `pkcs11-pro
 `tpm-provider`. Choose the providers you want to install depending on what is available on the
 platform.
 
-Create the Parsec socket directory.
+Create the `parsec` user.
 
 ```
-mkdir /tmp/parsec
+sudo useradd -m parsec
+sudo passwd parsec
 ```
 
-Create the `parsec-clients` group and set the correct permissions on the socket folder. Mutually
-trusted Parsec Clients will need to be in that group.
+Create the following Parsec directories, with good permissions.
+
+`/var/lib/parsec` for storing persistent data like the `mappings` folder. The service will run from
+here.
 
 ```
-sudo groupadd parsec-clients
-sudo chown :parsec-clients /tmp/parsec
-sudo chmod 750 /tmp/parsec
+sudo mkdir /var/lib/parsec
+sudo chown parsec /var/lib/parsec
+sudo chmod 700 /var/lib/parsec
 ```
 
-For example, adding the imaginary `parsec-client-1` user to the `parsec-clients` group:
+`/etc/parsec` to contain the configuration file.
 
 ```
-sudo usermod -a -G parsec-clients parsec-client-1
+sudo mkdir /etc/parsec
+sudo chown parsec /etc/parsec
+sudo chmod 700 /etc/parsec
 ```
 
-Users just added to that group might need to log-out and log-in again to make sure the change apply.
+`/usr/libexec/parsec` to contain the `parsec` executable binary file.
 
-Create and log in to a new user named `parsec`.
+```
+sudo mkdir /usr/libexec/parsec
+sudo chown parsec /usr/libexec/parsec
+sudo chmod 700 /usr/libexec/parsec
+```
+
+`/run/parsec` to contain the socket file. The `parsec-clients` group needs to be created. Mutually
+trusted Parsec Clients will need to be in that group.
+
+```
+sudo groupadd parsec-clients
+sudo mkdir /run/parsec
+sudo chown parsec:parsec-clients /run/parsec
+sudo chmod 750 /run/parsec
+```
+
+Log in to `parsec`.
 
 ```
-sudo useradd -m parsec
-sudo passwd parsec
 su --login parsec
 ```
 
 Depending on which features of Parsec the `parsec` user is going to use, it might need to be given
 more privileges in order to access some resources on the system. Refer to the
 [Providers](providers.md) page for more information.
 
-In its home directory, pull and install Parsec as a daemon. If a Rust toolchain is not available
-widely on the system, it will need to be [installed](https://www.rust-lang.org/tools/install) for
-that specific user.
+In its home directory, clone and compile Parsec. If a Rust toolchain is not available widely on the
+system, it will need to be [installed](https://www.rust-lang.org/tools/install) for that specific
+user.
+
+Below is an example with Parsec 0.5.0, update with the version you want!
 
 ```
-cargo install parsec-service --features $DESIRED_FEATURES
+git clone --branch 0.5.0 https://github.com/parallaxsecond/parsec
+cargo build --manifest-path parsec --features $DESIRED_FEATURES --release
+cp parsec/target/release/parsec /usr/libexec/parsec
```

Comment:
Could we suggest here using `caro install parsec-service --features $DESIRED_FEATURES --path /usr/libexec/parsec`?